Podfile

```diff
@@ -13,30 +13,14 @@ target 'TestingApp' do
 end
 
 target 'GliaWidgets' do
-  # Comment the next line if you don't want to use dynamic frameworks
-  use_frameworks!
```

Comment:
This defined [globally](https://github.com/salemove/ios-sdk-widgets/pull/316/files#diff-8f7d6adf31268a2d897ee34bd170592648d6e520aa237104395e4a4438af50cbL9), so we don't need to define once again